experimental/FTheoryTools/TateModels/attributes.jl

```diff
@@ -87,7 +87,7 @@ julia> tate_section_a6(t);
 Return the Tate polynomial of the global Tate model.
 
 ```jldoctest
-julia> t = global_tate_model(test_base())
+julia> t = su5_tate_model_over_arbitrary_3d_base()
 Global Tate model over a concrete base
```

Comment:
Doc tests fail, apparently this has a different output:
```suggestion
Global Tate model over a not fully specified base
```

Reply:
Yes, I am working on fixing the doctests right now. I hope that these changes will already speed-up the doctests, but there are other places which I hope to optimize.